### PR TITLE
Support 1.20.30

### PR DESCRIPTION
**Pull request by Konicai**
- BlockStateUpdater has been depended on to facilitate patching mappings for newer versions. I didn't use the `BlockStateUpdaters` class and instead configured my own context so that we can choose the version to update to (not just the max available).

### CODE REVIEW

**Comment by Konicai** [comment]
Tested on 1.20.0, 1.20.12, 1.20.30.25

**Comment by Konicai** [comment]
Requires https://github.com/CloudburstMC/BlockStateUpdater/pull/7

**Comment by Alemiz112** [comment]
Hi, good catch. I will check this out later today and then merge!

**Review by Camotoy — APPROVED** [review]

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -9,9 +9,10 @@ netty = "4.1.80.Final"
 guava = "29.0-jre"
 gson = "2.3.1" # Provided by Spigot 1.8.8
 websocket = "1.5.1"
-protocol = "3.0.0.Beta1-20230718.000033-102"
-protocol-connection = "3.0.0.Beta1-20230718.000033-101"
+protocol = "3.0.0.Beta1-20230908.171156-106"
+protocol-connection = "3.0.0.Beta1-20230908.171156-105"
 raknet = "1.0.0.CR1-20230703.195238-9"
+blockstateupdater="1.20.30-SNAPSHOT"
```
Comment (Camotoy) [inline gradle/libs.versions.toml:15]: Can you make this a static snapshot like Protocol?

**Pull request merged**